build/config/PublishData.json

```diff
@@ -28,6 +28,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.9p1"]
         },
+        "dev16.0.x-vs-deps": {
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.11.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev16.0", "dev16.0p1" ]
```

Comment:
Lets change the channels entry for master-vs-deps to be dev16.0p2 as well here.

Reply:
Actually I believe this channel should be _just_ 16.0p1 and dev16.0 goes on master-vs-deps too?

Reply:
Yes that's correct.